app/models/state_file_nj_intake.rb

```diff
@@ -154,8 +154,11 @@ class StateFileNjIntake < StateFileBaseIntake
   enum tenant_shared_rent_not_spouse: { unfilled: 0, yes: 1, no: 2}, _prefix: :tenant_shared_rent_not_spouse
   enum tenant_same_home_spouse: { unfilled: 0, yes: 1, no: 2}, _prefix: :tenant_same_home_spouse
 
+  def nj_gross_income
```

Comment:
[dust] do you need to call calculate here to make sure that happens before the medical expenses controller is reached?